core/src/main/scala/org/apache/spark/api/java/JavaPairRDD.scala

```diff
@@ -234,6 +234,9 @@ class JavaPairRDD[K, V](val rdd: RDD[(K, V)])
    * In addition, users can control the partitioning of the output RDD, the serializer that is use
    * for the shuffle, and whether to perform map-side aggregation (if a mapper can produce multiple
    * items with the same key).
+   *
+   * @note V and C can be different -- for example, one might group a RDD of type (Int, Int) into
+   * an RDD of type (Int, List[Int]).
```

Comment:
OK, it works alright as a standalone note. It could have stayed an inline sentence too. OK either way.